_lifeofinternatpuppet/post_19.md

```diff
@@ -19,13 +19,13 @@ Some of the key highlights from this week included:-
 
 - When doing the release prep for the puppetlabs-vcsrepo (v3.2.0), the tests were failing on Travis. I re-run the acceptance tests locally and they passed.
 
-More details in these [PRs]:-
-1.  https://github.com/puppetlabs/puppetlabs-vcsrepo/pull/486
-2.  https://github.com/puppetlabs/puppetlabs-vcsrepo/pull/487
+More details in these PRs:-
+1. [PR_1](https://github.com/puppetlabs/puppetlabs-vcsrepo/pull/486)
```

Comment:
PR_1 and PR_2 are not descriptive enough, please refer to these PRs as what they actually cover

Reply:
A new PR is raised :- https://github.com/puppetlabs/iac/pull/222